library/std/src/sync/once.rs

```diff
@@ -124,7 +124,7 @@ unsafe impl Sync for Once {}
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl Send for Once {}
 
-#[stable(feature = "ref_unwind_safe", since = "1.56.1")]
+#[stable(feature = "sync_once_ref_unwind_safe", since = "1.57.0")]
```

Comment:
This actually needs to be at least 1.59.0, since that is the current nightly version. It may be better to wait until this is ready to be merged to update the version though.

```suggestion
#[stable(feature = "sync_once_ref_unwind_safe", since = "1.59.0")]
```